src/installer/publish/prepare-artifacts.proj

```diff
@@ -88,13 +88,14 @@
     <PushToAzureDevOpsArtifacts
       ItemsToPush="@(ItemsToPush)"
       ManifestBuildData="@(ManifestBuildData)"
-      ManifestRepoUri="$(BUILD_REPOSITORY_URI)"
+      ManifestRepoUri="$(BUILD_REPOSITORY_NAME)"
```

Comment:
Does this repo have a new enough arcade to use ManifestRepoName instead of ManifestRepoUri?

Reply:
Master got a arcade 2 days ago -> https://github.com/dotnet/runtime/pull/41796

Also repo name is reflected in the manifests 

![image](https://user-images.githubusercontent.com/47157394/93270957-ffe2cd00-f766-11ea-9f2b-db2ca8c3b252.png)

![image](https://user-images.githubusercontent.com/47157394/93270997-18eb7e00-f767-11ea-9203-ccd0ef88385d.png)

![image](https://user-images.githubusercontent.com/47157394/93271018-256fd680-f767-11ea-9ca8-e37d89adf253.png)